R/PredictionClust.R

```diff
@@ -0,0 +1,44 @@
+#' @title Plot for PredictionClust
+#'
+#' @description
+#' Generates plots for [mlr3cluster::PredictionClust], depending on argument `type`:
+#' * `"scatter"` (default): scatterplot with correlation values and cluster assignments.
+#'
+#' @param object ([mlr3cluster::PredictionClust]).
+#' @param task ([mlr3cluster::TaskClust]).
+#' @template param_type
+#' @param ... (`any`):
+#'   Additional arguments, passed down to the respective `geom`.
+#'
+#' @return [ggplot2::ggplot()] object.
+#' @export
+#' @examples
+#' library(mlr3)
+#' library(mlr3cluster)
+#' library(mlr3viz)
+#'
+#' task = tsk("usarrests")
```

Comment:
This is not a mlr3 task, isn't it? CI also cannot find it.

Reply:
Just saw the discussion in MM. Let's clarify first the current state of mlr3cluster (with respect to the import of the package) and then we can proceed here.